docs/tutorials/dynamics_backend.rst

```diff
@@ -0,0 +1,489 @@
+            
+        schedules.append(schedule)
+        
+    job = backend.run(schedules, shots=100)
+    result = job.result()
+
+Visualize one of the schedules.
+
+.. jupyter-execute::
+
+    schedules[3].draw()
+
+Retrieve the counts for one of the experiments as would be done using the results object from a real
+backend.
+
+.. jupyter-execute::
+
+    result.get_counts(3)
+
+4. Simulating circuits at the pulse level using the :class:`.DynamicsBackend`
+-----------------------------------------------------------------------------
+
+For the :class:`.DynamicsBackend` to simulate a circuit, each circuit element must have a
+corresponding pulse schedule. These schedules can either be specified in the gates themselves, by
+attaching calibrations, or by adding instructions to the :class:`~qiskit.transpiler.Target`
+contained in the :class:`.DynamicsBackend`.
+
+4.1 Simulating circuits with attached calibrations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Build a simple circuit. Here we build one consisting of a single Hadamard gate on qubit :math:`0`,
+followed by measurement.
+
+.. jupyter-execute::
+
+    from qiskit import QuantumCircuit
+    
+    circ = QuantumCircuit(1, 1)
+    circ.h(0)
+    circ.measure([0], [0])
+    
+    circ.draw("mpl")
+
+Next, attach a calibration for the Hadamard gate on qubit :math:`0` to the circuit. Note that here
+we are only demonstrating the mechanics of adding a calibration; we have not attempted to calibrate
+the schedule to implement the Hadamard gate with high fidelity.
+
+.. jupyter-execute::
+
+    with pulse.build() as h_q0:
+        pulse.play(
+            pulse.library.Gaussian(duration=256, amp=0.2, sigma=50, name="custom"),
+            pulse.DriveChannel(0)
+        )
+    
+    circ.add_calibration("h", [0], h_q0)
+
+Call run on the circuit, and get counts as usual.
+
+.. jupyter-execute::
+
+    %time res = backend.run(circ).result()
+    
+    res.get_counts(0)
+
+4.2 Simulating circuits via gate definitions in the backend :class:`~qiskit.transpiler.Target`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Alternatively to the above work flow, add the above schedule as the pulse-level definition of the
+Hadamard gate on qubit :math:`0` to `backend.target`, which impacts how jobs are transpiled for the
+backend. See the :class:`~qiskit.transpiler.Target` class documentation for further information.
+
+.. jupyter-execute::
+
+    from qiskit.circuit.library import HGate
+    from qiskit.transpiler import InstructionProperties
+    
+    backend.target.add_instruction(HGate(), {(0,): InstructionProperties(calibration=h_q0)})
+
+Rebuild the same circuit, however this time we do not need to add the calibration for the Hadamard
+gate to the circuit object.
+
+.. jupyter-execute::
+
+    circ2 = QuantumCircuit(1, 1)
+    circ2.h(0)
+    circ2.measure([0], [0])
+    
+    %time result = backend.run(circ2).result()
```

Comment:
Does it work with 2+ qubits? I think it internally calls pulse scheduler but there is a [known bug](https://github.com/Qiskit/qiskit-terra/issues/9488).

Reply:
You also need to properly configure the meas_map (backend.configuration().meas_map) to schedule, i.e. if you specify `[[0,1]]` in the meas_map you need to provide a schedule to measure q0 and q1 simultaneously (this is a typical setting in IBM backend).

Reply:
I didn't want to make this limitation. A user can configure that if they would like to.

Reply:
Yes. I'm fine with current code as long as it works.